OJS.Workers.ExecutionStrategies/CSharp/DotNetCore/DotNetCoreCompileExecuteAndCheckExecutionStrategy.cs

```diff
@@ -149,5 +167,32 @@ private void CreateRuntimeConfigJsonFile(string directory, string text)
 
             File.WriteAllText(jsonFilePath, text);
         }
+
+        private void AddUsingNamespacesForDotNetSix<T>(IExecutionContext<T> executionContext)
+        {
+            if (this.Type != ExecutionStrategyType.DotNetCore6CompileExecuteAndCheck)
+            {
+                return;
+            }
+
+            var stringBuilder = new StringBuilder();
+
+            this.DotNetSixDefaultUsingNamespaces
+                .ForEach(usingNamespace => this.AddUsingNamespaceIfNotExist(usingNamespace, stringBuilder, executionContext.Code));
+
+            stringBuilder.AppendLine(executionContext.Code);
+
+            executionContext.Code = stringBuilder.ToString();
+        }
```

Comment:
Having the change method `GetNamespaceIfNotExist()`, this can be done as:
```
executionContext.Code = string.Join(
    Environment.NewLine,
    this.DotNetSixDefaultUsingNamespaces.Select(ns => this.GetNamespaceIfNotExist(ns, executionContext.Code)
)
```